README.md

```diff
@@ -36,7 +36,8 @@ Specifically, it loads configuration from:
     * `CONJUR_CORE_URL`
     * `CONJUR_AUTHN_URL`
  * A username and api key can be read from `~/.netrc` if stored there by
-    `conjur authn login`
+    `conjur authn login`. If so, `CONJUR_AUTHN_LOGIN` and `CONJUR_API_KEY` are
+    not required.
```

Comment:
Perhaps this should also mention that the netrc file will *only* be checked if login and api key are not provided otherwise, since this is a new behavior?